kolibri/plugins/learn/assets/src/vue/learn-page/allcontent.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+
+  <card-grid :header="'All Content'" v-if="all.content.length">
+
+    <div slot="headerbox" class="allnav">
+
```

Comment:
:100: for the effort, but in this case I'd go for `p` instead of `label` (it's visually hidden anyways) - less convoluted, not sure how would `label` render here in a screenreader.

Or just go with `aria-label` from the above suggestion!